### ISSUE TITLE
Lag issues

### ISSUE THREAD

**Issue by 5ambito** [tsunamayo/Starship-EVO#4672]
### Short Description of the issue:

When looking at a star from any distance a large drop in FPS is caused, when looking away from the star FPS returns to normal. When inside a ship this is not an issue.

### Steps to reproduce the issue:

Unsure, this is the first time it has happened and only on a specific world I am playing.

### Environment
- Starship EVO version: early access 22w14a
- Graphics Card: GTX1050

### Additional information:

_(Screenshots, Videos, Error logs, Related issues)_

Please also include the `Player.log` file. You can find the log file here: `%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\`

If you issue concerns a particular design please include any blueprint that could help to reproduce the issue.
Blueprint folder is over at:
`%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\Save_Data\Blueprints\`
You will need to zip the folder to upload it. 
To do so Right click and choose [Send to -> compressed zip file] then drag the .zip onto your report to upload it.
![unknown](https://user-images.githubusercontent.com/103335082/162589945-abd7af73-48ed-4cbe-bc83-9fe776aedd6a.png)

**Comment by tsunamayo** [comment]
@5ambito Hi, thanks for booking the issue.
Could you please send the log file and the character file as specified in the case so I can investigate further?
Thanks

**Comment by ProPeach** [comment]
I actually have a similar issue, however the fps drop occurs when I look at the gas giant, not the star.
This is a new world in yesterdays patch
https://user-images.githubusercontent.com/27857711/162594946-75066173-fefc-40d0-9b74-37afc922b5ec.mp4

[Player.log](https://github.com/tsunamayo/Starship-EVO/files/8457921/Player.log)
[Planet Lag.zip](https://github.com/tsunamayo/Starship-EVO/files/8457923/Planet.Lag.zip)

**Comment by ProPeach** [comment]
Oddly, in this older world the fps drop does not occur
[Warp 2.zip](https://github.com/tsunamayo/Starship-EVO/files/8457929/Warp.2.zip)
[Player.log](https://github.com/tsunamayo/Starship-EVO/files/8457930/Player.log)

However this world does have the issue occur (it was made in round #4 of the novox campaign
[Player.log](https://github.com/tsunamayo/Starship-EVO/files/8457934/Player.log)
[Voxel4.zip](https://github.com/tsunamayo/Starship-EVO/files/8457935/Voxel4.zip)

On each of these I'm just looking  at the nearby gas giant and then looking away to show the difference. If you sit your player out a little further than the ice rings then you should reproduce. Closer than that the effect is lesser

**Comment by tsunamayo** [comment]
@ProPeach Hi, I cant reproduce, how big of a FPS drop do you get? Also do you think it could be because you are looking at the space station (even if you dont really see them, are they inside the screen).
Thanks

**Comment by tsunamayo** [comment]
@ProPeach Okay so I made some test, and although I dont really get any perf impact out of this there is definitely something a bit fishy with the far cam. So I will pull the update out for now, I keep you guys posted!

**Comment by tsunamayo** [comment]
@5ambito @ProPeach Okay so I managed to fix the issue. I re-uploaded the build with the fix, with the same build id. Thanks for the quick report! Cheers

**Comment by 5ambito** [comment]
Ok no problem, I'll have to do it later today. How do I send the log to
you? This is my first issue report so im unsure as to how I send it to you.

**Comment by ProPeach** [comment]
@tsunamayo great! I've tested those worlds on my end with the new patch and the issue is completely gone, thanks for the quick fix!
For completions sake regarding the questions, The drop I experienced was a capped 60 down to around 40 as per the video, really I should have tried with v-sync off to see the full drop, whoops. Perhaps the drop wasn't visible to you if you had v-sync on with your 3060? The space station wasn't in frame in that Warp 2 world either, nor were any other brick based entities in frame.
Either way, case closed o/
